src/pages/cloud-development/index.md

```diff
@@ -3,8 +3,8 @@ title: Cloud Development
 ---
 ## Cloud Development
 
-Cloud developoment is the usage of software engineering practices on infrastructure components in a shared resource pool. This can be done in multiple manners, within your company or organizations own data center, or in a "public" shared datacenter owned and operated by a third party.
+Cloud development is the usage of software engineering practices on infrastructure components in a shared resource pool. This can be done in multiple manners, within your company or organizations own data center, or in a "public" shared datacenter owned and operated by a third party.
 
-Popular choices for Public Cloud providers are Amazon Web Services, Microsoft Azure, and Google Compute Engine. These providers operate clusters of massive datacenters whereby thousands of different individuals, companies, and organizations consume these resources. Operaitng the Cloud involves enabling the partitioning of these shared resources for the usage of these different users. The consumption of the resources is tracked (also referred to as metered) and then billed back to the user depending on the different financial scheme for the various compute, network, and storage resources consumed.
+Popular choices for Public Cloud providers are Amazon Web Services, Microsoft Azure, and Google Compute Engine. These providers operate clusters of massive datacenters whereby thousands of different individuals, companies, and organizations consume these resources. Operatng the Cloud involves enabling the partitioning of these shared resources for the usage of these different users. The consumption of the resources is tracked (also referred to as metered) and then billed back to the user depending on the different financial scheme for the various compute, network, and storage resources consumed.
```

Comment:
~~Operatng the Cloud~~ Operating the Cloud

Reply:
I would use `processing` instead of `compute` in the last sentence